plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergModule.java

```diff
@@ -61,8 +61,8 @@ public void configure(Binder binder)
         binder.bind(ConnectorPageSinkProvider.class).to(IcebergPageSinkProvider.class).in(Scopes.SINGLETON);
         binder.bind(ConnectorNodePartitioningProvider.class).to(IcebergNodePartitioningProvider.class).in(Scopes.SINGLETON);
 
-        configBinder(binder).bindConfig(OrcReaderConfig.class);
-        configBinder(binder).bindConfig(OrcWriterConfig.class);
+        configBinder(binder).bindConfig(OrcReaderConfig.class, "iceberg");
+        configBinder(binder).bindConfig(OrcWriterConfig.class, "iceberg");
```

Comment:
This is a backward-incompatible change. Is it intentional?

Reply:
> This is a backward-incompatible change. Is it intentional?

[Praveen2112](https://github.com/Praveen2112) suggested using connector name as the prefix to avoid confusion

Reply:
As for parameters, we need to unify whether to use the original method or use the prefix of connector name. @findepi @Praveen2112

Reply:
I'm not sure if its is a `backward-incompatible change` as all OrcReader/OrcWriter config has a prefix `hive` and we have extracted it.

Reply:
I discussed this with @electrum .

`OrcReaderConfig` was already bound in Iceberg, so a user could have configured eg `hive.orc.max-buffer-size`.
With this change, such configuration would be rejected for Iceberg.

I am not saying this is a wrong direction, but it definitely requires more thought to ensure backwards compatibility.
We should not do this within this PR, **please revert these changes here**.

BTW there is a somewhat more import problem here. `OrcReaderConfig` provides eg `hive.orc.use-column-names`, a setting that is meaningless for Iceberg, it's Hive-only.
To cleanup these configs we would perhaps need to split these classes.

Reply:
I agree with this.. Initially I was a bit worried on having configuration property with `hive` as a prefix - which might confuse user when we configure it for `iceberg` and since most of the Orc related config has `hive` as a prefix - I thought we could extract and inject according to the connector implementation. We can also remove `hive` prefix and treat them as a legacy config so that we could have a unifying config across hive and iceberg.